reversion/revisions.py

```diff
@@ -106,35 +117,51 @@ def _pop_frame():
 
 
 def is_manage_manually():
```

Comment:
should probably be called `is_managed_manually`

Reply:
I want change original implementation..